.travis.yml

```diff
@@ -7,8 +7,9 @@ python:
 # - "3.5" # ecdsa tests fail in 3.5.
   - "3.6"
   - &latest_py3 "3.7"
+  - "3.8"
```

Comment:
Are you aware that now `latest_py3` no longer points to the latest Python 3?

Reply:
Yeah, I did that on purpose.  python3.8 is so new that I didn't trust `twine` and the deployment steps to work in that environment.  My not-great metric is that py3.8 required code updates for JP itself, and we're not even that big of a library, so I opted to hold that back for stability purposes.

We'll update that eventually, though.  Thanks for the heads-up.

Reply:
I think it would be easy to change the variable name `latest_py3` to something less mis-leading in the mean time. Just an idea.